Minesweeper.py

```diff
@@ -322,10 +337,10 @@ def place_mines(self):
                 num_mines -=1
 
     def uncover(self, pos):
+        """ uncovers tiles when they are clicked and check for win or lose conditions when the tiles are uncovered
+        """
```

Comment:
still haven't implemented a generator function for getting all tiles. That would make your code cleaner.